src/main/java/com/example/todolist/API/Restful/RestController/TaskController.java

```diff
@@ -0,0 +1,40 @@
+package com.example.todolist.API.Restful.RestController;
+
+import com.example.todolist.API.Restful.Dto.Base.Response;
+import com.example.todolist.API.Restful.Dto.Request.CreateTaskRequest;
+import com.example.todolist.Domain.Entity.Task;
+import com.example.todolist.Domain.Service.TaskService;
+import com.example.todolist.Infrastructure.Repository.TaskRepository;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpSession;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    @Autowired
+    private TaskService taskService;
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @PostMapping
+    public ResponseEntity<Response> createTask(@RequestBody @Valid CreateTaskRequest createTaskRequest, HttpServletRequest httpServletRequest) {
+        HttpSession session = httpServletRequest.getSession(false);
+        String userId = (String) session.getAttribute("user_id");
```

Comment:
can we have a better way to write these lines of code? this could be duplicated everywhere